test/test_payu.py

```diff
@@ -222,8 +223,8 @@ def test_parse_ldd_output():
     with open(ldd_output_path, 'r') as f:
         ldd_output = f.read()
     required_libs = payu.fsops.parse_ldd_output(ldd_output)
-    assert(len(required_libs), 4) 
-    assert(required_libs['libmpi.so.40'], '/apps/openmpi/4.0.2/lib/libmpi.so.40')
+    assert(len(required_libs) == 4) 
+    assert(required_libs['libmpi.so.40'] == '/apps/openmpi/4.0.2/lib/libmpi.so.40')
```

Comment:
This is a fix to tests from #351? 

Oops! Sorry I missed those when reviewing.

Reply:
Yes they are, I should've picked up on it sooner as it was picked up on the test warnings..